eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -62,6 +62,17 @@ case class Features(activated: Set[ActivatedFeature], unknown: Set[UnknownFeatur
 
   def hasPluginFeature(feature: UnknownFeature): Boolean = unknown.contains(feature)
 
+  def areSupported(remoteFeatures: Features): Boolean = {
```

Comment:
```suggestion
  /**
  * NB: this method is not reflexive, see [[Features.areCompatible]].
  */
  def areSupported(remoteFeatures: Features): Boolean = {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1576/commits/c026fa00a7c9e2223760850eae220f810829067b